webui/react/src/services/apiBuilder.ts

```diff
@@ -47,7 +47,7 @@ export const processApiError = (name: string, e: Error): void => {
     silent,
     type: isAuthError ? ErrorType.Auth : ErrorType.Server,
   });
-  throw e;
+  return e;
```

Comment:
**suggestion:** I think there's more value in returning the `DaError` that gets returned by the `handleError` function right above and only return `DaError` types. The caller already has access to the `Error` type error, because they are passing it in.